api/courses.go

```diff
@@ -39,7 +39,9 @@ func configGinCourseRouter(router *gin.Engine) {
 	adminOfCourseGroup.POST("/presets", updatePresets)
 	adminOfCourseGroup.POST("/deleteLectures", deleteLectures)
 	adminOfCourseGroup.POST("/renameLecture/:streamID", renameLecture)
+	adminOfCourseGroup.POST("/updateLectureSeries/:streamID", updateLectureSeries)
 	adminOfCourseGroup.POST("/updateDescription/:streamID", updateDescription)
+	adminOfCourseGroup.POST("/deleteLectureSeries/:streamID", deleteLectureSeries)
```

Comment:
It doesn't change behaviour, however, I think an update should be the HTTP Method PUT, and a delete DELETE. The URL the could be something like `/lectureSeries/:streamID`.

Reply:
yes thats better ! 👍